### ISSUE TITLE
Improve parked domains detection 

### ISSUE THREAD

**Issue by jarelllama** [jarelllama/Scam-Blocklist#304]
- [x] Test 16 runs in parallel instead of 14
- [x] Assume domains that errored during curl remain parked during unparked check
- [x] Test curl max-time of 2 instead of 3
- [x] Code review
- [x] Add additional test cases to test_functions.sh
- [x] Check results after full run
